internal/storage/minio_chunk_manager.go

```diff
@@ -106,6 +107,12 @@ func newMinioChunkManagerWithConfig(ctx context.Context, c *config) (*MinioChunk
 	return mcm, nil
 }
 
+// SetVar set the variable value of mcm
+func (mcm *MinioChunkManager) SetVar(ctx context.Context, bucketName string) {
```

Comment:
The naming is weird, and the function doesn't seem necessary

Reply:
it's just for a easier test.